app/src/controllers/administrate/projects/sagas.js

```diff
@@ -85,6 +86,8 @@ function* addProject({ payload: projectName }) {
       projectRole: PROJECT_MANAGER,
       entryType: PROJECT_TYPE_INTERNAL,
     };
+    const organizationSlug = yield select(projectOrganizationSlugSelector);
+    const projectKey = yield select(projectKeySelector);
```

Comment:
In this case, previously we use projectName for this action.
But now, we have no information about projectKey and organizationSlug, because the user cannot enter this.
And here I dont now how to figure it out correctly.
In my opinion when we add new project, we must create 'PUT' request, and response must contains projectKey and OrganizationSlug for this projrct. Or we must add fields in to our modal window, like 'projectKey' and 'organizationSlug'